### ISSUE TITLE
Is this package doing the right thing?

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/AffineArithmetic.jl#27]
In standard affine arithmetic (`FullAffine`), each new interval is supposed to introduce a new "noise term".

Also, currently we assume that we only want one noise term per "independent variable". This is not necessarily correct.

**Comment by mewilhel** [comment]
I think the Messine's AF1 & AF2 forms are probably the most computationally tractable., at least for the applications I'm interested in... So I'm happy with the behavior. It may be good to add a subtypes of FullAffine which use the AF1 (one error term), and AF2 (one positive, one negative, and one mixed-sign error term).

**Comment by dpsanders** [comment]
Yes I think the current behaviour is a good compromise, but of course suggestions are very welcome.
